presto-google-sheets/src/main/java/io/prestosql/plugin/google/sheets/SheetsMetadata.java

```diff
@@ -139,6 +140,9 @@ public ConnectorTableMetadata getTableMetadata(ConnectorSession session, Connect
         if (!schemaName.isPresent()) {
             throw new PrestoException(SHEETS_UNKNOWN_SCHEMA_ERROR, "Schema not present - " + schemaName);
         }
+        if (!listSchemaNames().contains(schemaName.get())) {
```

Comment:
I think we should explicitly filter out information_schema.

In fact, i think `listTables` should not be called with schemaName=information_schema, but that may require more changes. Currently, each connector needs to take care of this on its own.